vmdb/config/locales/en.yml

```diff
@@ -940,6 +948,8 @@ en:
       ems_folders:                 Folders
       ems_infra:                   Infrastructure Provider
       ems_infras:                  Infrastructure Providers
+      ems_container:               Provider
```

Comment:
I'm wondering whether we need to call this "Containers Provider" or just Provider is suffice.
Containers Provider is more aligned with infra/cloud providers.
(we can always change it later)